buildSrc/build.gradle.kts

```diff
@@ -20,5 +21,10 @@ gradlePlugin {
       id = "strumenta.multiplatform"
       implementationClass = "com.strumenta.kotlinmultiplatform.gradle.plugins.StrumentaMultiplatformModulePlugin"
     }
+
+    register("strumentaGradlePlugin") {
```

Comment:
What does it do?

Reply:
It's yet another internal convention plugin, so that `antlr-kotlin-gradle-plugin` doesn't have to configure itself, but it only needs to apply `id("strumenta.gradle.plugin")`. It's a way to centralize modules' setup like the other two you see there.